blinkit/styles/styles.css

```diff
@@ -0,0 +1,335 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    font-family:Arial, Helvetica, sans-serif;
+}
+nav{
+    display: flex;
+}
+img {
+    display: inline-block;
+    width: 178px;
+    height: 75px;
+    /* border-right: 1px solid grey; */
+    
+}
+
+#delivery-time{
+    padding: 5vh;
+    font-size: 18px;
+    font-weight: bold;
+}
+
+input.nosubmit {
+    border: none;
+    box-shadow: 3px 3px 3px 3px lightgray;
+    border-radius: 0.5rem;
+    width: 50vw;
+    height: 7vh;
+    padding: 9px 4px 9px 40px;
+    margin-top: 3vh;
+     background: transparent url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' class='bi bi-search' viewBox='0 0 16 16'%3E%3Cpath d='M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z'%3E%3C/path%3E%3C/svg%3E") no-repeat 13px center;
+  }
+  .btn-navbar{
+    display: flex;
+    width: 25%;
+    justify-content: space-evenly;
+
+  }
+  #Login{
+    font-weight: 500;
+    font-size: 25px;
+    height: 52px;
+    width: 112px;
+    text-align: center;
+    padding-top: 7px; 
+    margin: auto;
+    
+  }
+  
+  #Login a{
+
+    
+    text-decoration: none;
+    color: black;
+
+
+  }
+   
+
+
+  #my-cart {
+    background-color: rgb(12, 131, 31);
+    border-radius: 8px;
+    cursor: pointer;
+    height: 52px;
+    width: 112px;
+    box-shadow: none;
+    border: none;
+    font-size: 18px;
+    color: white;
+    font-weight: 300;
+    margin:auto;
+  }
+
+  .menu-1 ul{
+    display: flex;
+    list-style-type: none;
+    justify-content: center;
+    font-size: 14px;
+    border-top: solid 0.5px lightgrey;
+    border-bottom: solid 0.5px lightgrey;
+  }
+
+  .menu-1-list li {
+    margin: 15px;
+  }
+
+  .menu-1-list div:hover{
+    cursor: pointer;
+    background-color: lightgrey;
+  }
+
+  .main-body{
+    display: flex;
+    justify-content: center;
+  }
+
+  .side-menu > div{
+    display: flex;
+    justify-content: left;
+    padding: 12px;    
+    position: relative;
+    align-items: center;
+    gap: 12px;
+    background-color: rgb(255, 255, 255);
+    border-left: 1px solid rgb(238, 238, 238);;
+    border-right: 1px solid rgb(238, 238, 238);
```

Comment:
DRY -- Don't repeat yourself
For better quality of code, we should not repeat code.
I can see the same properties being used in the css of the .grid-header.
Please try to come up with a solution for this problem here and let's discuss about it